README.md

```diff
@@ -1 +1,137 @@
-# ruby-template
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Catalog Of My Things
+
+## _Microverse Ruby Group Capstone Project_
+
+> This is an educational project to create a console app that you to keep a record of different types of things you won: books, music albums, movies, and games. Everything will be based on the UML class diagram presented below.
+
+![catalog_of_my_things](https://user-images.githubusercontent.com/98527559/182126086-fc6a58be-ce66-43cc-9a67-42ddf9633100.png)
+
+## Learning Objectives 🔖
+
+- [x] Insert and query data in SQL.
+- [x] Use primary key & foreign key mechanisms for joining tables.
+- [x] Understand the different types of relationships between tables.
+- [x] Query multiple tables.
+- [x] Prepare complex queries that answer analytical questions.
+- [x] Run a program using the command line.
+- [x] Use Ruby syntax for basic programming operations.
+- [x] Apply Ruby best practices and language style guides in code.
+- [x] Describe the SOLID principles of OOP.
+- [x] Implement classes and objects in Ruby.
+- [x] Understand the four main principles of OOP.
+- [x] Implement encapsulation and inheritance with Ruby.
+- [x] Create a UML class diagram.
+- [x] Explain the difference between associations, aggregations, and composition in OOP.
+- [x] Set up associations between classes and objects.
+- [x] Store data in files.
+- [x] Build interactive console apps.
+- [x] Recognize the value of making equal contributions to group projects to produce the best outcome.
+
+## Project Documentation 📄
+
+- [x] Here is the presentation video [link](https://drive.google.com/file/d/1Fi1Og-ZkgPK-UnpXF65DkXFkomawnbAF/view?usp=sharing)👈
+
+## Built With 🛠️
+
+- [x] Programming Language: [Ruby](https://www.ruby-lang.org/en/)
+- [x] Linter: [Rubocop](https://rubocop.org/)
+- [x] VS Code
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Ruby.
+
+You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+
+```
+$ ruby --version && irb
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone git@github.com:ger619/ruby-capstone.git
+```
+
```

Comment:
- [ ] It would be better if we add instructions about how to run the app in the README file. Here is a quick fix `ruby main.rb` but before that in order not to run into errors like I did, I had to rename the `MusicAlbum`  to `musicalbum` from the `display_menu` file

Reply:
Thanks for your review. We added some instructions on how to run the application as well as a note that the `data` folder will be automatically generated once you exit the application. This will trigger to save all the data that was created on the app. Also, when you enter the app, it will read these generated `.json` files inside the `data` folder and add the items on the current session.